packages/rax/src/vdom/composite.js

```diff
@@ -33,7 +33,12 @@ function handleError(instance, error) {
   }
 
   if (boundary) {
-    boundary.componentDidCatch(error);
+    if (boundary._internal) {
```

Comment:
会有 boundary._internal 为 null 情况吗？

Reply:
有 当unmount的时候

Reply:
合并为这样，没有 _internal 的时候，让error throw出来，会不会更好理解些

```suggestion
    if (boundary  && boundary._internal) {
```

或者在这里加注释说明下，免得后面维护的同学会认为这里是个bug，漏了处理没有 _internal 的场景

Reply:
> boundary && boundary._internal
@chenjun1011 感觉不太对，没有 _internal 的时候，我们只是没必要处理这个报错，因为react处理报错的模型是驱动boundary更新变成不报错的视图，既然boundary都没有了，为何还需要更新？